incubator/kafka/values.yaml

```diff
@@ -272,7 +296,7 @@ zookeeper:
   persistence:
     enabled: false
     ## The amount of PV storage allocated to each Zookeeper pod in the statefulset
-    # storage: "2Gi"
+    # size: "2Gi"
```

Comment:
This key was incorrectly specified in the original values.yaml, should be "size", not "storage"

Reply:
👍